qutebrowser/utils/jinja.py

```diff
@@ -127,3 +130,33 @@ def render(template, **kwargs):
 
 environment = Environment()
 js_environment = jinja2.Environment(loader=Loader('javascript'))
+
+
+@functools.lru_cache()
+def template_config_variables(template: str) -> typing.FrozenSet[str]:
```

Comment:
Please add tests for this in `tests/unit/utils/test_jinja.py` - this should be quite easy to test, and `jinja.py` is marked as one of the files which have 100% test coverage.